### PR TITLE
PROGRAMMES-7090 - Jenkins to CodeBuild

### PR DESCRIPTION
**Pull request by Bobkidbob**
https://jira.dev.bbc.co.uk/browse/PROGRAMMES-7090

### CODE REVIEW

File: script/build [hunk 1]
```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+
+set -e
+RESULT=0
+
+# If we're not already using php70+, try and solve that
+if ! (php --version | grep -q "PHP 7") ; then
```
Comment (benolot) [inline script/build]: Everything else is 7.1, shouldn't this be too?

File: buildspec.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+version: 0.2
+
+env:
+    parameter-store:
+        OAUTH: "OAUTH_TOKEN"
+
+phases:
+    build:
+        commands:
+            - script/build
```
Comment (benolot) [inline buildspec.yml]: Don't forget to check this is executable

Comment (Bobkidbob) [reply buildspec.yml]: I forgot.

File: script/build [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+# setup and run composer
+if [ -n "$OAUTH" ]; then
+    composer config -g github-oauth.github.com $OAUTH
+fi
+
+composer --no-ansi --no-interaction install
+
+# Make sure the reports directory exists
+if [ ! -d "build/reports" ]; then
```
Comment (Tasiobg) [inline script/build]: We don't use "build/reports" any more in codebuil, feel free to remove lin 21, 22 and 23

File: script/build [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    mkdir -p build/reports
+fi
+
+# Show php version for debugging
+php --version
```
Comment (Tasiobg) [inline script/build]:
```suggestion

```
You can remove this unless you have any particular reason to keep it, is juts printing the version of PHP

File: script/build [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+# Run PHPUnit
+vendor/bin/phpunit || RESULT=1
+
+# Run PHPCS
+vendor/bin/phpcs --report=checkstyle --report-file=build/reports/checkstyle.xml || RESULT=1
```
Comment (Tasiobg) [inline script/build]:
```suggestion
vendor/bin/phpcs || RESULT=1
```
Reports files are not used

**Pull request merged**